test/unit/app/utils/test_kenya_state_machine.py

```diff
@@ -2,22 +2,30 @@
 from functools import partial
 from faker.providers import phone_number
 from faker import Faker
+import json
 
 from helpers.user import UserFactory
 from helpers.ussd_session import UssdSessionFactory
 from server.utils.ussd.kenya_ussd_state_machine import KenyaUssdStateMachine
+from server.models.user import User
+
+from server.utils import user as user_utils
 
 fake = Faker()
 fake.add_provider(phone_number)
 # why do i get dupes if i put it directly on standard_user...?
 phone = partial(fake.msisdn)
 
-standard_user = partial(UserFactory)
+unactivated_user = partial(UserFactory, is_activated=False)
+standard_user = partial(UserFactory, pin_hash=User.salt_hash_secret('0000'), failed_pin_attempts=0)
```

Comment:
since we don't have `pin=` anymore, this is how we can preset pins in test. does this seem correct @enjeyw?

Reply:
yeah totally legit methinks.